package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
The study template has been updated since this study was started, at the very least we need to update `rally.js` from `0.3` to the latest (`0.6`), so it points to the correct site when the core add-on is not found:

https://www.npmjs.com/package/@mozilla/rally

Reply:
Rally is now updated to 0.6.0